Shared/ContentView.swift

```diff
@@ -10,19 +10,19 @@ import SwiftUI
 struct ContentView: View {
     @State private var checkAmount = 0.0
     @State private var numberOfPeople = 0
-    @State private var tipPercentage = 10
+    @State private var tipPercentage = 0
     @FocusState private var amountIsFocused: Bool
     
     var tipPercentages = [0, 10, 15, 20, 25]
-    
-    var totalPerPerson: Double {
-        let peopleCount = Double(numberOfPeople + 2)
+    var checkTotal: Double{
```

Comment:
Let's leave a space between Double and {

Reply:
Resolved